src/proteus/dswx_hls.py

```diff
@@ -1744,21 +1747,66 @@ def _compute_and_apply_cloud_layer(wtr_2_layer, fmask,
         logger.info(error_msg)
         raise Exception(error_msg)
 
-    logger.info(f'mask adjacent to cloud/cloud-shadow mode:'
-                f' {mask_adjacent_to_cloud_mode}')
- 
     # Check Fmask cloud shadow bit (3) => bit 0
-    cloud_layer[np.bitwise_and(fmask, 2**3) == 2**3] = 1
+    preliminary_cloud_layer[np.bitwise_and(fmask, 2**3) == 2**3] = 1
 
     if mask_adjacent_to_cloud_mode == 'mask':
         # Check Fmask adjacent to cloud/shadow bit (2) => bit 0
-       cloud_layer[np.bitwise_and(fmask, 2**2) == 2**2] = 1
+       preliminary_cloud_layer[np.bitwise_and(fmask, 2**2) == 2**2] = 1
 
     # Check Fmask cloud bit (1) => bit 2
-    cloud_layer[np.bitwise_and(fmask, 2**1) == 2**1] += 4
+    preliminary_cloud_layer[np.bitwise_and(fmask, 2**1) == 2**1] += 4
 
-    # If cloud (1) or cloud shadow (3), mark WTR as WTR_CLOUD_MASKED
-    wtr_layer[cloud_layer != 0] = WTR_CLOUD_MASKED
+    return preliminary_cloud_layer
+
+
+
+def _add_snow_to_cloud_layer(wtr_2_layer, cloud_layer, fmask,
+                             mask_adjacent_to_cloud_mode):
+    """Finish computing the CLOUD layer by adding the snow/ice class
+       to the CLOUD layer.
+       This function succeeds the function _compute_preliminary_cloud_layer()
+       and preceeds _apply_cloud_masking()
+
+       Parameters
+       ----------
+       wtr_2_layer: numpy.ndarray
+            Cloud-unmasked interpreted water layer
+       cloud_layer : numpy.ndarray
+            Preliminary cloud mask (without the snow/ice class)
```

Comment:
Something like:

```suggestion
            Preliminary cloud mask (without the snow/ice class)
            with dtype of uint8 and values assigned as follows:
                0: Not masked
                1: Cloud shadow or adjacent to cloud/cloud shadow
                4: Cloud
                5: Cloud and class 1 (cloud shadow or adjacent to cloud/cloud shadow)
                255: Fill value (no data)
```

Reply:
Just update the docstrings. Thank you, @nemo794